tests/interactive/test_app.py

```diff
@@ -467,3 +471,46 @@ async def test_passes():
         assert isinstance(app.current_module, ModuleOp)
         # Assert that the current module has been changed accordingly
         assert app.current_module.is_structurally_equivalent(expected_module)
+
+
+@pytest.mark.asyncio()
+async def test_argument_pass_screen():
+    """Test that clicking on a pass that requires passes opens a screen to specify them."""
+    async with InputApp().run_test() as pilot:
+        app = cast(InputApp, pilot.app)
+        # clear preloaded code and unselect preselected pass
+        app.input_text_area.clear()
+
+        await pilot.pause()
+        # Testing a pass
+        app.input_text_area.insert(
+            """
+        func.func @hello(%n : i32) -> i32 {
+  %two = arith.constant 0 : i32
+  %res = arith.addi %two, %n : i32
+  func.return %res : i32
+}
+        """
+        )
+        app.passes_tree.root.expand
+        await pilot.pause()
+
+        root_children = app.passes_tree.root.children
+        distribute_stencil_node = None
+
+        for node in root_children:
+            assert node.data is not None
+            pass_val, _ = node.data
+            if pass_val.name == stencil_global_to_local.DistributeStencilPass.name:
+                distribute_stencil_node = node
+
+        assert distribute_stencil_node is not None
+
+        # When running in a test, node selection does not send the expected event
+        # app.passes_tree.select_node(distribute_stencil_node)
```

Comment:
remove commented code

Reply:
It's kind of apart of the comment, I clarified it